app/assets/styles/styles.css

```diff
@@ -35,156 +71,270 @@ body {
-
-}
-img {
-	max-width: 100%;
-	height: auto;
-}
-
-a {
-	text-decoration: none;
-}
+	font-size: 0.85em;
 
-p {
-	line-height: 1.55;
 }
 
 /*Positioning and sizing the main content, setting it to be inside 75% of the browser width. */
 .post-wrapper {
-  overflow: hidden;
-  padding-left: 18px;
-  padding-right: 18px;
-  max-width: 75vw;
-  margin: 80px auto;
-  font-size: 1.15rem;
+	overflow: hidden;
+	padding-left: 18px;
+	padding-right: 18px;
+	font-size: 1rem;
+	max-width: 75vw;
+	margin: 80px auto;
 }
+
 .post__our-journey h1 {
 	font-family: Cambria, Cochin, serif;
-	font-size: 4.25rem;
+	font-size: 2.6rem;
 	font-style: italic;
 	font-weight: 100;
-	margin: 0.25rem;
 	/*margin-bottom: 0.67rem;*/
 }
- .post__our-journey--our-story h1  {
-	margin-bottom: 1rem;}
+
+.post__our-journey--our-story h1 {
+	margin-bottom: 1rem;
+}
 
 .post__our-journey h2 !important {
-	font-size: 2rem;
+	font-size: 1rem;
 	font-weight: 300;
 	margin-top: 0;
 	margin-bottom: 0.2rem;
 }
 
-.post {
-	display: grid;
-	grid-gap: 10px 50px;
-	grid-template-columns: 2fr 12fr 6fr;
-}
-
-.post >* {
-	grid-column: 1 /-2;
-}
-
 .post__our-journey > blockquote {
-	grid-column: 1/-1;
-	font-size: 2.5rem;
+	font-size: 1.5rem;
 	font-style: italic;
 	text-align: center;
 	margin: 0;
 }
- .post-side {
-	background: #FAFAFA;
-	padding: 10px;
-	grid-row: span 2;
-	align-self: center;
+
+.post> blockquote {
+	font-size: 1.5rem;
+	font-style: italic;
+	text-align: center;
+	margin: 0;
 }
 
-.post-side--right {
-	grid-column: span 1/ -1;
-	border-left: 2px solid #ffc600;
+.post-albums__scholar {
+	/*background: rgba(255, 255, 255, 0.2);*/
+	box-shadow: 0 0 5px rgba(254, 178, 54, 0.3);
+	color: #3b3a30;
+	font-weight: 150;
 }
 
-.post-side--right p {
-	line-height: 1.50;
+img {
+	max-width: 100%;
+	height: auto;
 }
 
-.post>figure {
-	margin: 0;
-	grid-column: 1/ -1;
+a {
+	text-decoration: none;
 }
 
-figcaption {
-	font-size: 13px;
-	text-align: center;
-	margin-bottom: 0.67rem;
+p {
+	line-height: 1.55;
 }
-.post> blockquote {
-	grid-column: 1/-1;
-	font-size: 2.5rem;
-	font-style: italic;
-	text-align: center;
-	margin: 0;
+
+.post-side {
+	background: #FAFAFA;
+	padding: 10px;
+	align-self: center;
 }
 
 /*Styling the album with CSS-grid. Defining a different wrapper for the album. */
 .albums-wrapper {
-  background-color: #eaece5;
-  overflow: hidden;
-  padding: 1.25rem;
-  max-width: 85vw;
-  font-size: 1.15rem;
-  margin-left: auto;
-  margin-right: auto;
+	background-color: #eaece5;
+	overflow: hidden;
+	padding: 1.25rem;
+	max-width: 85vw;
+	margin-left: auto;
+	margin-right: auto;
 }
 
-.post-albums {
-	display: grid;
-	grid-template-columns: repeat(auto-fit, minmax(400px, 1fr));
-	grid-gap: 1.25rem;
+@media only screen and (min-width: 595px) {
+	.home-hero h1 {
+		font-size: 2.4em;
+	}
+
+	.home-hero h2 {
+		font-size: 0.89em;
+	}
+
+	.home-hero .btn {
+		margin-top: 20px;
+		width: 150px;
+		padding: 0.70em;
+		border-radius: 5px;
+		font-size: 0.93em;
+	}
```

Comment:
It is best to group all the code related to a component together, including all the media queries. Since home-hero is defined above, it would be best to move this to where you have those styles.